pkg/lib/serverTime.js

```diff
@@ -547,14 +772,9 @@ function ChangeSystimeBody({ state, errors, change }) {
                         <Validated errors={errors} error_key="manual_time" />
                     </div>
                 }
-                { mode == "ntp_time_custom" &&
-                    <Validated errors={errors} error_key="ntp_servers">
-                        <div id="systime-ntp-servers-row">
-                            <div id="systime-ntp-servers">
-                                { ntp_servers }
-                            </div>
-                        </div>
-                    </Validated>
+                { mode == "ntp_time" &&
+                <NTPConfig state={ntp_config} change={() => change("ntp_config", ntp_config)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20230113-125341-c8c3ecb6-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)